libvast/src/logger.cpp

```diff
@@ -27,6 +26,8 @@
 #include <spdlog/sinks/rotating_file_sink.h>
 #include <spdlog/sinks/syslog_sink.h>
 
```

Comment:
One additional request: Please rebase and adjust the first commit message to use active voice present tense. A good guideline is that `When applied, this commit will <commit-message>` should be a full sentence without a stop at the end.

Additionally, please apply the same rule to the story title.

Reply:
The story title in Shortcut is different but follows the rule.